Test failures, possible caused by GAP.jl 0.7.1

Seeing these failures [in ci](https://github.com/oscar-system/Oscar.jl/runs/4047784113?check_suite_focus=true) for PR #744 (which is just modifying docs, so shouldn't affect regular tests at all):
```
Test Summary:              | Pass  Total
Other automorphisms groups |    2      2
Test Summary: | Pass  Total
Transitivity  |   22     22
ERROR: Package Oscar errored during testing (received signal: KILL)
Stacktrace:
 [1] pkgerror(msg::String)
   @ Pkg.Types /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.6/Pkg/src/Types.jl:55
 [2] test(ctx::Pkg.Types.Context, pkgs::Vector{Pkg.Types.PackageSpec}; coverage::Bool, julia_args::Cmd, test_args::Cmd, test_fn::Nothing)
   @ Pkg.Operations /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.6/Pkg/src/Operations.jl:1693
 [3] test(ctx::Pkg.Types.Context, pkgs::Vector{Pkg.Types.PackageSpec}; coverage::Bool, test_fn::Nothing, julia_args::Cmd, test_args::Cmd, kwargs::Base.Iterators.Pairs{Symbol, IOContext{Base.PipeEndpoint}, Tuple{Symbol}, NamedTuple{(:io,), Tuple{IOContext{Base.PipeEndpoint}}}})
   @ Pkg.API /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.6/Pkg/src/API.jl:343
 [4] test(pkgs::Vector{Pkg.Types.PackageSpec}; io::IOContext{Base.PipeEndpoint}, kwargs::Base.Iterators.Pairs{Symbol, Bool, Tuple{Symbol}, NamedTuple{(:coverage,), Tuple{Bool}}})
   @ Pkg.API /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.6/Pkg/src/API.jl:80
 [5] test(; name::Nothing, uuid::Nothing, version::Nothing, url::Nothing, rev::Nothing, path::Nothing, mode::Pkg.Types.PackageMode, subdir::Nothing, kwargs::Base.Iterators.Pairs{Symbol, Bool, Tuple{Symbol}, NamedTuple{(:coverage,), Tuple{Bool}}})
   @ Pkg.API /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.6/Pkg/src/API.jl:96
 [6] top-level scope
   @ none:1
Error: Process completed with exit code 1.
Run doctests
```

I am suspecting this is caused by GAP.jl 0.7.1 :-(. I have absolutely no idea *how* at this point, but I'll look at it. However, I have to head home now. I am posting this here so people know what's going on. 

Any ideas welcome. I'll try to fix this tonight, at worst gotta yank that version :-(

--- a/src/Groups/libraries/perfectgroups.jl
+++ b/src/Groups/libraries/perfectgroups.jl
@@ -28,7 +28,7 @@ function perfect_group(::Type{T}, n::Int, m::Int) where T <: GAPGroup
    return G
 end
 
-perfect_group(n::Int, m::Int) = perfect_group(FPGroup,n,m)
+perfect_group(n::Int, m::Int) = perfect_group(PermGroup,n,m)
 
 """
     perfect_identification(G::GAPGroup)

--- a/test/Groups/libraries.jl
+++ b/test/Groups/libraries.jl
@@ -39,8 +39,8 @@ end
    @test isperfect(G)
    @test !isperfect(symmetric_group(5))
 
-   @test perfect_group(120,1) isa FPGroup
-   @test perfect_group(PermGroup,120,1) isa PermGroup
+   @test perfect_group(120,1) isa PermGroup
+   @test perfect_group(FPGroup,120,1) isa FPGroup
    @test_throws ArgumentError perfect_group(MatrixGroup,120,1)
 
    @test isisomorphic(perfect_group(60,1),G)[1]

